WordPress/Classes/Services/LoginFacade.m

```diff
@@ -68,17 +71,28 @@ - (void)signInToSelfHosted:(LoginFields *)loginFields
             if ([options objectForKey:@"wordpress.com"] != nil) {
                 [self signInToWordpressDotCom:loginFields];
             } else {
+                NSString *versionString = options[@"software_version"][@"value"];
```

Comment:
This feels gross to copy and paste the code here since it's nearly identical to the code in BlogService (which seriously needs to get removed). If you don't think that refactor is easy lets just open an issue and get that TODO solved finally! :grin: